package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ern-container-publisher-maven",
-  "version": "1.0.5",
+  "version": "1.0.6",
```

Comment:
Please remove the version update commit from this branch. We'll do the version/publish flow independent from this PR.